control/modelsimp.py

```diff
@@ -463,77 +477,80 @@ def markov(Y, U, m=None, transpose=False):
 
     """
     # Convert input parameters to 2D arrays (if they aren't already)
-    Umat = np.array(U, ndmin=2)
-    Ymat = np.array(Y, ndmin=2)
 
-    # If data is in transposed format, switch it around
-    if transpose:
-        Umat, Ymat = np.transpose(Umat), np.transpose(Ymat)
-
-    # Make sure the system is a SISO system
-    if Umat.shape[0] != 1 or Ymat.shape[0] != 1:
-        raise ControlMIMONotImplemented
+    # Get the system description
+    if (len(args) < 1):
```

Comment:
Parenthesis are not needed here.  The more standard style convention would be
```
if len(args) < 1:
```

Reply:
Done.